python/tvm/target/target.py

```diff
@@ -510,23 +548,27 @@ def validate_hvx_length(codegen_hvx, sim_args):
                 + cpu_attr["post"]
             )
 
-        return sim_cpu + " " + validate_hvx_length(hvx, sim_args)
+        return sim_cpu + " " + validate_hvx_length(hvx, sim_options)
+
+    # LLVM options string
+    def create_llvm_options(cpu_ver, args):  # pylint: disable=unused-argument
```

Comment:
Can we simply pass `llvm_options` as the argument here, since we use nothing out of that scope. `args` is a bit more vague.